lib/coffee-autocompile.coffee

```diff
@@ -19,9 +19,18 @@ class StylusAutocompile
     @compile()
 
   compile: ->
-    params = @getParams @activeEditor.getURI(), @activeEditor.getText()
-    return unless params?
-
+    text = @activeEditor.getText()
+    firstComment = text.match /^\s*#\s*(.*)\n*/
```

Comment:
This doesn't seem to check for a hashbang (`#!...`) that *must* be on the first line anymore